test/create-snowpack-app/create-snowpack-app.test.js

```diff
@@ -15,28 +16,28 @@ const format = (stdout) =>
 
 describe('create-snowpack-app', () => {
   // test npx create-snowpack-app bin
-  it('npx create-snowpack-app', async () => {
+  it('npx create-snowpack-app', () => {
     const template = 'app-template-preact'; // any template will do
+    const installDir = path.resolve(__dirname, 'test-install');
+
+    rimraf.sync(installDir);
 
     // run the local create-snowpack-app bin
-    await execa(
+    execa.sync(
       'node',
       [
         './packages/create-snowpack-app',
         `./test/create-snowpack-app/test-install`,
         '--template',
         `../../../packages/@snowpack/${template}`,
         '--use-yarn', // we use Yarn for this repo
-        '--force', // saves you from having to manually delete things
```

Comment:
Unrelated to this PR: I was seeing this test consistently time out. I’d be OK with removing it, because I don’t think it’s that valuable as we’re testing the template builds.

But in the interest of science, I am consistently seeing it run faster by using Yarn’s local cache. The changes to this test should speed up CI by a bit, but in case anyone wants to remove this test altogether I think it’d be fine too.